documentation/docs/components/outputs/command-outputs/nym-node-node-details.md

```diff
@@ -1,3 +1,3 @@
 ```sh
-[2m2024-10-29T09:51:44.459681Z[0m [31mERROR[0m [2mnym-node/src/config/upgrade_helpers.rs[0m[2m:[0m[2m16[0m[2m:[0m Failed to finish upgrade - failed to load config file using path '/home/m/.nym/nym-nodes/default-nym-node/config/config.toml'. detailed message: No such file or directory (os error 2)
+[2m2024-11-06T12:18:46.742880Z[0m [31mERROR[0m [2mnym-node/src/config/upgrade_helpers.rs[0m[2m:[0m[2m17[0m[2m:[0m Failed to finish upgrade - failed to load config file using path '/home/m/.nym/nym-nodes/default-nym-node/config/config.toml'. detailed message: No such file or directory (os error 2)
```

Comment:
this seems it did something else then wished for?

Reply:
Yeah isn't being caught by the error grabbing in the script it seems...

Reply:
If we're not using this anywhere then I think we can ignore for the moment, I need to spend some time doing an overhaul of autodoc in the near future anyway